examples/multi_keyring.cpp

```diff
@@ -48,9 +43,7 @@ void encrypt_or_decrypt(
     if (!aws_cryptosdk_session_is_done(session)) abort();
     if (input_consumed != input_len) abort();
 
-    /* This destroys both the session and the CMM, but not the keyring, since the
-     * keyring pointer was not released.
-     */
+    /* This destroys both the session but not the keyring, since the keyring pointer was not released. */
```

Comment:
"This destroys the session and not the keyring.."

Reply:
Good catch. I missed that.